dash_auth/auth.py

```diff
@@ -26,6 +27,9 @@ def _protect_views(self):
                 self.app.server.view_functions[view_name] = \
                     self.auth_wrapper(view_method)
 
+    def is_authorized_hook(self, func):
```

Comment:
It would be clearer to implement these hooks as an optional parameter to the `__init__` method.